third_party/pcsc-lite/naclport/server_clients_management/src/permissions_checking/user-prompting-checker-unittest.js

```diff
@@ -254,85 +258,133 @@ goog.exportSymbol(
         null /* expectedStorageDataToBeWritten */,
         MockedDialogBehavior.USER_CANCELS /* mockedDialogBehavior */,
         function(userPromptingChecker) {
-          return negatePromise(userPromptingChecker.check(FAKE_CLIENT_1_ID));
+          return negatePromise(
+              userPromptingChecker.check(FAKE_CLIENT_1_ORIGIN));
         }));
 
 goog.exportSymbol(
     'test_UserPromptingChecker_AlreadyInStorage_OneItem',
     makeTest(
-        {[STORAGE_KEY]:
-             {[FAKE_CLIENT_1_ID]: true}} /* fakeInitialStorageData */,
+        {
+          [STORAGE_KEY]: {[FAKE_CLIENT_1_ORIGIN]: true}
+        } /* fakeInitialStorageData */,
+        null /* expectedStorageDataToBeWritten */,
+        MockedDialogBehavior.NOT_RUN /* mockedDialogBehavior */,
+        function(userPromptingChecker) {
+          return userPromptingChecker.check(FAKE_CLIENT_1_ORIGIN);
+        }));
+goog.exportSymbol(
```

Comment:
nit: I think each test should be preceded by an empty line (here and in a few other places in this file)